draft-ietf-httpbis-priority.md

```diff
@@ -428,11 +428,10 @@ The Priority HTTP header field carries priority parameters (see {{parameters}}).
 It can appear in requests and responses. It is an end-to-end signal of the
 request priority from the client or the response priority from the server.
 {{merging}} describes how intermediaries can combine the priority information
-from client requests and server responses to correct or amend the precedence.
-Clients cannot interpret the appearance or omission of a Priority response
-header as acknowledgement that any prioritization has occurred. Guidance for how
-endpoints can act on Priority header values is given in {{server-scheduling}}
-and {{client-scheduling}}.
+sent from clients and servers. Clients cannot interpret the appearance or
+omission of a Priority response header as acknowledgement that any
```

Comment:
```suggestion
omission of a Priority response header field as acknowledgement that any
```
this was always there but since you changed it above, we can be consistent.

P.S how did we miss all of these missing fields? :D